src/alr/alr-commands-init.adb

```diff
@@ -82,6 +82,9 @@ package body Alr.Commands.Init is
       procedure Generate_Gitignore;
       --  Generate or append .gitignore
 
+      procedure Generate_Workspace;
```

Comment:
Since this is for VSCode only, I would rename the procedure into something like `Generate_VSCode_Workspace`.